app/models/declaration/submissions/SubmissionStatus.scala

```diff
@@ -47,6 +47,10 @@ object SubmissionStatus extends Enumeration {
     case UNKNOWN                         => "Unknown status"
   }
 
+  def formatOnDeclInfoPages(status: SubmissionStatus): String = status match {
+    case ADDITIONAL_DOCUMENTS_REQUIRED => "Documents Required"
+    case _                             => format(status)
+  }
```

Comment:
Should this (and the format method above) not return i18n message keys rather than text to render on a view? Display logic don't belong in the data model.

Reply:
You are absolutely right, but I think we need a proper tech ticket for that. I do not think should be done within this ticket.

Reply:
Going this route I'd ask if `format` method should not also be mmoved from this place to some view-dedicated class/object?